app/components/stacks-by-organization/component.js

```diff
@@ -0,0 +1,24 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  organizationGroups: function() {
+    var groups = Ember.A([]);
+
+    this.get('stacks').forEach(function(stack) {
+      var currentOrganizationId = stack.get('_data.links.organization');
+      var currentOrganization = stack.get('organization');
+
+      if(!groups.findBy('organizationId', currentOrganizationId)) {
+        groups.addObject(Ember.Object.create({
+          organizationId: currentOrganizationId,
+          organization: currentOrganization,
+          stacks: Ember.A([])
+        }));
+      }
+
+      groups.findBy('organizationId', currentOrganizationId).get('stacks').addObject(stack);
+    });
```

Comment:
Unfortunately not. For a non-collection resource (i.e. a singular one) we don't store the links in meta.